app/views/layout.html

```diff
@@ -1,14 +1,50 @@
+{%- from 'action-link/macro.njk' import actionLink %}
+{%- from 'back-link/macro.njk' import backLink %}
+{%- from 'breadcrumb/macro.njk' import breadcrumb %}
+{%- from 'button/macro.njk' import button %}
+{%- from 'card/macro.njk' import card %}
+{%- from 'character-count/macro.njk' import characterCount %}
+{%- from 'checkboxes/macro.njk' import checkboxes %}
+{%- from 'contents-list/macro.njk' import contentsList %}
+{%- from 'date-input/macro.njk' import dateInput %}
+{%- from 'details/macro.njk' import details %}
+{%- from 'do-dont-list/macro.njk' import list %}
+{%- from 'error-message/macro.njk' import errorMessage %}
+{%- from 'error-summary/macro.njk' import errorSummary %}
+{%- from 'fieldset/macro.njk' import fieldset %}
+{%- from 'footer/macro.njk' import footer %}
+{%- from 'header/macro.njk' import header %}
+{%- from 'hero/macro.njk' import hero %}
+{%- from 'hint/macro.njk' import hint %}
+{%- from 'images/macro.njk' import image %}
+{%- from 'input/macro.njk' import input %}
+{%- from 'inset-text/macro.njk' import insetText %}
+{%- from 'label/macro.njk' import label %}
+{%- from 'pagination/macro.njk' import pagination %}
+{%- from 'panel/macro.njk' import panel %}
+{%- from 'radios/macro.njk' import radios %}
+{%- from 'select/macro.njk' import select %}
+{%- from 'skip-link/macro.njk' import skipLink %}
+{%- from 'skip-link/macro.njk' import skipLink %}
+{%- from 'summary-list/macro.njk' import summaryList %}
+{%- from 'tables/macro.njk' import table %}
+{%- from 'tabs/macro.njk' import tabs %}
+{%- from 'tag/macro.njk' import tag %}
+{%- from 'task-list/macro.njk' import taskList %}
+{%- from 'textarea/macro.njk' import textarea %}
+{%- from 'warning-callout/macro.njk' import warningCallout %}
```

Comment:
These lines aren't strictly needed, as you could import the relevant components on each page (and the service manual website includes these within its code snippets), but they make it easier to use the components without having to remember to import things - also keeps the code neater.